tests/configure/test_config.py

```diff
@@ -78,3 +81,38 @@ def test_command(): # noqa
                                              TEST_PROFILE_2])
     assert '--profile can only be provided once. The profiles [{}, {}] were provided.'.format(
         TEST_PROFILE_1, TEST_PROFILE_2) in result.output
+
+
+TEST_HOST = 'https://test.cloud.databricks.com'
+TEST_TOKEN = 'testtoken'
+
+
+def test_command_headers():
+    @click.group()
+    @config.profile_option
+    def test_group():
+        pass
+
+    @click.group()
+    @config.profile_option
+    def test_group_2():
```

Comment:
Instead of `test_group` and `test_group_2` could you use some better names?

Reply:
sure